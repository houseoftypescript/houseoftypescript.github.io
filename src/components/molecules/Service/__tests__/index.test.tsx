import { render } from '@testing-library/react';
import { Service } from '../index';
import { StaticImageData } from 'next/image';

describe('Service', () => {
  it('renders correctly', () => {
    const button = render(<Service image={{} as StaticImageData}></Service>);
    expect(button.container).toMatchSnapshot();
  });
});
