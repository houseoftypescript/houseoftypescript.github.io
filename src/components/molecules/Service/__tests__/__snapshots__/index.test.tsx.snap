// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Service renders correctly 1`] = `
<div>
  <div
    class="grid grid-cols-1 sm:grid-cols-3 gap-x-0 gap-y-8 md:gap-8"
  >
    <div
      class="col-span-1"
    >
      <img
        alt=""
        class="w-16 h-16 mx-auto"
        data-nimg="1"
        decoding="async"
        loading="lazy"
        src=""
        style="color: transparent;"
      />
    </div>
    <div
      class="col-span-2"
    >
      <div
        class="text-center sm:text-left"
      >
        <div
          class="mb-4"
        >
          <h3
            class="font-bold text-md sm:text-lg"
          />
        </div>
        <p />
      </div>
    </div>
  </div>
</div>
`;
